stacks-signer/CHANGELOG.md

```diff
@@ -11,10 +11,25 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 ### Changed
 
+## [3.0.0.0.3.0]
+
+### Added
+
+### Changed
+
 - Allow a miner to extend their tenure immediately if the winner of the next tenure has committed to the wrong parent tenure (#5361)
 
+## [3.0.0.0.2.0]
```

Comment:
Are we doing two releases? Wouldn't tenure_last_block_proposal_timeout_secs be part of 3.0.0.03.0?

Reply:
this was the previous release, it was missing from develop branch so I've added it to this pr (and then the release branch will pr back to develop)